src/invertObject.js

```diff
@@ -19,7 +19,22 @@
  * @return {Object}
  */
 function invertObject(items) {
-  // write code here
+  let sameValue = '';
```

Comment:
you can solve this task without this variable.

Reply:
Done